references.bib

```diff
@@ -288,6 +297,15 @@ @inproceedings{Wails2024a
 	discussion_url = {https://github.com/net4people/bbs/issues/312},
 }
 
+@inproceedings{Ding2023a,
```

Comment:
I'm undecided about adding this. Steganography work is certainly adjacent to censorship but I see it as its own, fairly substantial, body of research and I'd like to avoid adding more and more steganography work over time. What do you think? Is there merit in adding this particular paper?

Reply:
Ok, fair. Let's get this merged then. Thanks again for your help!